src/pcrlock/pcrlock.c

```diff
@@ -3701,6 +3702,7 @@ static int verb_lock_uki(int argc, char *argv[], void *userdata) {
 
 static int event_log_reduce_to_safe_pcrs(EventLog *el, uint32_t *pcrs) {
         _cleanup_free_ char *dropped = NULL, *kept = NULL;
+        bool dropped_relevant_pcr = false;
```

Comment:
i think it would be nicer if this was a mask of pcrs, not a boolean

Reply:
k. not needed for the purpose at hand but could be done for style/consistency reasons for sure